packages/shared/src/components/CommentFeed.tsx

```diff
@@ -103,10 +103,7 @@ export default function CommentFeed<T>({
         isFetchingNextPage={queryResult.isFetchingNextPage}
         canFetchMore={checkFetchMore(queryResult)}
         fetchNextPage={queryResult.fetchNextPage}
-        className={classNames(
-          'w-full',
-          isNewMobileLayout && isMainFeed && 'px-4',
```

Comment:
What should this look like in control values? Can you also attach screenshot of the old UI? Just to make sure everything still looks normal.